ecl2df/gruptree.py

```diff
@@ -225,10 +223,44 @@ def edge_dataframe2dict(dframe):
 
     children, parents = zip(*edges)
     roots = set(parents).difference(children)
-    trees = []
-    for root in list(roots):
-        trees.append({root: subtrees[root] for root in roots})
-    return trees
+    return [{root: subtrees[root]} for root in sorted(roots)]
+
+
+def _sub_tree_from_dict(nested_dict, name):
+    """Internal recursive function for generating trees."""
+    assert isinstance(nested_dict, dict)
+    tree = treelib.Tree()
+    tree.create_node(name, name)
+    child_names = list(nested_dict.keys())
+    child_names.sort()
+    for child_name in child_names:
+        # The paste() function requires unique node names.
+        tree.paste(name, _sub_tree_from_dict(nested_dict[child_name], child_name))
+    return tree
+
+
+def tree_from_dict(nested_dict):
+    """Convert a dictonary to a treelib Tree.
+
+    The treelib representation of the trees is used
+    for pretty-printing (in ASCII) of the tree, you
+    only need to do str() of the returned result
+    from this function.
+
+    See `https://treelib.readthedocs.io/`
+
+    Args:
+        nested_dict: nested dictonary representing a tree.
+
+    Returns:
+        treelib.Tree
+    """
+    if not nested_dict:
```

Comment:
The truthyness of python would allow this function to successfully run, although we might consider that it should fail.

Here we're allowing e.g. `tree_from_dict([])`, `tree_from_dict(0)` and `tree_from_dict(None)` to all give an empty string, although we explicitly say the argument should be a dictionary.

It's a bit nitpicky, but I figured the `if not nested_dict` was covering a bit much.